dasharo-stability/m2-wifi.robot

```diff
@@ -93,14 +94,40 @@ Suite Teardown      Run Keyword
 #    END
 #    Exit from root user
 
-SMW0004.001 Wi-fi connection after suspension (Ubuntu 22.04)
+SMW004.001 Wi-fi connection after suspension (Ubuntu 22.04)
     [Documentation]    Check whether the Wi-Fi card is detected and working
     ...    correctly after performing suspension.
-    Skip If    not ${M2_WIFI_SUPPORT}    SUD004.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SUD004.001 not supported
+    Skip If    not ${M2_WIFI_SUPPORT}    SMW004.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SMW004.001 not supported
+    Skip If    ${PLATFORM_SLEEP_TYPE_SELECTABLE}    SMW004.001 not supported
```

Comment:
I do not get it, now we have 3 cases, where one is with S3, one with S0ix, one with what?